arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -225,12 +225,12 @@
             "$ref": "#/parameters/RegistryNameParameter"
           },
           {
-            "name": "registryUpdateParameters",
+            "name": "registry",
             "in": "body",
             "description": "The parameters for updating a container registry.",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/RegistryUpdateParameters"
+              "$ref": "#/definitions/Registry"
```

Comment:
Thank you, it should be good now. On a side note, once we switch to Registry -> PUT -> Registry model with `CreateOrUpdate`, a separate PATCH operation for updating resources seems to be redundant ...